.github/ISSUE_TEMPLATE/incident-response.md

```diff
@@ -0,0 +1,55 @@
+---
+name: Incident response
+about: Use to track the steps that should happen during an incident, such as a data breach or unexpected downtime, even if it's just a suspicion.
+---
+
+_**Do not put sensitive information in this issue.** Use a file on Google Drive with access restricted._
+
+**Severity:** High/Medium/Low
+
+Delegate tasks where possible.
+
+## Initiate
+
+1. [x] Create an issue from this template.
+1. [ ] Declare an incident in the relevant Slack channel, such as [#benefits-general][benefits-general].
+   - Include brief details about the concern.
+1. [ ] Start a video call.
+1. [ ] Share a link to the video call in Slack, asking relevant parties to join.
+
+## Assess
+
+- [ ] Determine the impact.
+- [ ] Assign the severity above:
+  - **High:** Possible/confirmed breach of sensitive information, such as production system credentials or personally-identifiable information (PII)
+  - **Medium:** Full Benefits downtime for more than 30 minutes
+  - **Low:** Partial service degredation
```

Comment:
Open to feedback on these. Any triggers I'm missing?

Reply:
I think this list does a good job of covering the most likely triggers. Some others that I can think of are: data loss and zero-day vulnerabilities.